script/Ciel/Day 10/10_01_CIEL10_4_3.txt

```diff
@@ -387,92 +414,105 @@ Yeah... Setting that aside―――
 }
 [sha:5658a540b35338c9add76455fc508c1ca27f9f4c]{
 -- Page 269, Offset 27798.
 -- この際ですからはっきりと忠告……いえ、警告しておきましょう。現在、この街で最も危ういのは彼女です。それは彼女が吸血鬼だから、という事ではありません。
-This is a good opportunity for me to advise you... no, to warn you. As we speak, there is none more dangerous in this city than her. This is not due to the fact that she is a vampire.
+If you're going to be involved,
+ then let me give you some advice... no, let me give you a warning.
+ As things stand, that woman is the most dangerous creature in this city.
+ That's not just because she's a vampire.
 }
 [sha:b6812623c84b0c45a65a7fe689cc2f4453d0225a]{
 -- Page 269, Offset 27799.
 -- 今の彼女はかつてないほど異常な状態です。下手に刺激すれば炸裂する爆弾と言ってもいい」
-She is currently behaving more abnormally than at any time in the past. You could liken her to a bomb that will violently explode if badly handled."
+The way she's behaving right now is utterly unprecedented.
+ She's like a bomb that could go off in your face if mishandled."
 }
 [sha:81018ee6dccfb951fa93ae31313b5dc4e9bd88f8]{
 -- Page 270, Offset 27800. Mods: @n.
 -- 「……爆弾、
-"A bomb―――
+"She's like...
 }
 [sha:c65af4a9b822f7c7154b0ded512e2f6aec8d89f1]{
 -- Page 270, Offset 27801. Glued.
 -- ですか？
-You say?
+ a bomb?
 }
 [sha:e2794aa2062eb1e5436c02b38283e44c854afd4a]{
 -- Page 270, Offset 27802.
 -- そりゃあアイツは何しでかすか分からないヤツですから、その表現は正しいですけど……でも、別に……」
-I don't know what kind of things she's done, so that might not be a wrong description... But, she's not really―――"
+// Possibly a bit strong for 何しでかすか分からないヤツ?
+I mean,
+ she %{i}is%{/i} an unpredictable walking disaster,
+ so that's honestly not an unreasonable description of her, but...
+ she's not exactly―――"
 }
 [sha:16e6d16ddc85f49d19d84cd946717623a1cda88e]{
 -- Page 271, Offset 27803.
 -- 危うい、なんて事はない筈だ。
-A danger or anything like that, I think.
+She's not exactly %{i}dangerous%{/i}, right?
 }
 [sha:3fc44f272fc3588317a15122569c675c09fc75b6]{
 -- Page 271, Offset 27804.
 -- アイツは意味のない行動はしない。その力はすべて吸血鬼を倒すことに使われていた。
-She won't act senselessly. Her power is entirely dedicated to defeating vampires.
+She doesn't do anything without a good reason to,
+ and the only thing she's done with her powers is defeat vampires.
 }
 [sha:3dbb5612d523dee3fc91c11bf54685dd54b0934a]{
 -- Page 272, Offset 27805.
 -- 「どうでしょうか。今の彼女は自分の感情に従って活動しています。
-"I wouldn't be so sure. As she currently is, she behaves under the influence of her emotions.
+"I wouldn't be so sure.
+ Right now, her actions are completely driven by her emotions.
 }
 [sha:7422c921dd3df7b32a34a85ad32a6da729f45daf]{
 -- Page 272, Offset 27806.
 -- これまでは真祖としての<条約|コード>に従っていたのに、その条約を破りかけている」
-Until now, she's followed a code as a True Ancestor and is now about to break this pact."
+Up to this point she's always followed the True Ancestor's code,
+ but she's suddenly decided to break that pact."
 }
 [sha:9bc18b46851436d2a8c7ac52cb2636f145be491f]{
 -- Page 273, Offset 27807.
 -- 「……アレは無邪気な子供そのものです。
-"She's exactly like a naïve child.
+"She's the very picture of a naïve child.
```

Comment:
I think "the very picture of" is overstating things. Innocent child is the much more common collocation.
>She is essentially an innocent child.

Reply:
Innocent's primary meaning shines through more as 'not guilty', rather than 'knows not what they do', which is I think more the aim here.

Reply:
Innocent in the collocation with child almost never means "not guilty". It means they are child-like and pure.